modules/govuk/manifests/apps/mapit.pp

```diff
@@ -52,6 +52,7 @@
         # https://docs.djangoproject.com/en/1.8/ref/contrib/gis/install/geolibs/
         'binutils',
         'libproj-dev',
+        'postgresql-server-dev-9.3',
```

Comment:
Have you considered doing this through the postgresql module?

Reply:
This is in essence just installing the package. There is no need to use the module since we want to keep dependencies at a minimum.

Reply:
One of the reasons I'm mentioning the module is that I'm currently looking at changing the version of PostgreSQL in the development-vm [1]. If PostgreSQL packages are installed in more places, without using a common version, this will potentially make managing it more complex.

1: https://github.com/alphagov/govuk-puppet/pull/8240